Sources/Foundation/NSStringAPI.swift

```diff
@@ -632,6 +632,7 @@ extension StringProtocol {
 
     /// Returns a representation of the string as a C string
     /// using a given encoding.
+    @available(*, deprecated, message: "On platforms without Objective-C autorelease pools, use withCString(encodedAs:_) instead")
```

Comment:
This one is actually safe to use though, right?

Reply:
Not unless we reimplement it to use a lifetime-limited version of the `NSString` `cString(using:)` function. Which I suppose we could do.

Reply:
I don't know if it's worth it, but it might mean less churn.